src/components/Profile/ProfileUserDetails/ProfileUserDetails.js

```diff
@@ -35,18 +35,23 @@ export class ProfileUserDetails extends Component {
           firstName, lastName, username, email, bio, image,
         },
       } = this.props;
+      let image1;
+      if (image) {
+        // eslint-disable-next-line no-unused-expressions
+        image.split(':')[0] === 'https' ? image1 = image : image1 = `https://res.cloudinary.com/djxhcwowp/image/upload/v${image}`;
```

Comment:
Why did you disable linting for this line? It looks like there's a variable that is unused.

Reply:
No, the issue was that whole line was receiving an error. All the variables are used in that. that is why I put this.